experimental/alp/alp/library/blas.py

```diff
@@ -0,0 +1,135 @@
+typest = """
```

Comment:
Same comment re using the harness and metaprogramming IR from python in the future: this is really hacky (fine for a first commit).